maintainers/maintainer-list.nix

```diff
@@ -5147,6 +5147,12 @@
       fingerprint = "FC1D 3E4F CBCA 80DF E870  6397 C811 6E3A 0C1C A76A";
     }];
   };
+  exploitoverload = {
```

Comment:
This change should go to a separate commit in this PR. See https://nixos.org/manual/nixpkgs/stable/#var-meta-maintainers